src/main/kotlin/command/Ping.kt

```
@@ -17,12 +19,16 @@ class Ping(private val jda: JDA) : Command() {
         return CommandCategory.ESSENTIAL
     }
 
-    override fun handler(event: MessageReceivedEvent) {
-        event.channel.sendMessage("Pong!").queue()
+    override suspend fun handler(event: MessageReceivedEvent) {
+        val message = try { event.channel.sendMessage("Pong!").await() } catch (e: Exception) { throw CommandExitException("Failed to send message") }
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)